ciao-controller/tables/workload_storage.csv

```diff
@@ -0,0 +1 @@
+79034317-3beb-447e-987d-4e310a8cf410,"",1,1,0,"image",73a86d7e-93c0-480e-9c41-ab42f69b7799
```

Comment:
Would be helpful here if the commit message described what this line means.  I think it means declare that volume uuid 79034317-3beb-447e-987d-4e310a8cf410 will be a copy of the image uuid 73a86d7e-93c0-480e-9c41-ab42f69b7799?  If the first '1' is bootable and the second '1' is persistent, this would be a workload-specific read-write volume and something would enforce that only one instance of this workload would be allowed in order to prevent two instances writing to the same volume?